README.md

```diff
@@ -52,12 +65,12 @@ In that case you will have to see if the error is something that needs to be fix
 
 The automatic variant will skip dependency download and install if the download was installed previously. If a dependency install failed for some reason or you cancelled it, you will have to manually install that package (see the downloads directory).
 
+------
 # Further Information
 
 - see [structure.md](structure.md) for an overview of how the project is organized
 - see [https://nexus-mods.github.io/vortex-api](https://nexus-mods.github.io/vortex-api/) for a description of the extension api
 - see [https://wiki.nexusmods.com/index.php/Vortex](https://wiki.nexusmods.com/index.php/Vortex) for usage information
-- run "yarn run doc" the create an html page from code documentation
```

Comment:
this seems to be a deprecated script as its no longer in the `package.json`. Please do correct me if I'm wrong!